package.json

```diff
@@ -2,11 +2,14 @@
   "private": true,
   "scripts": {
     "build": "vitruvius build --src src --dest lib --ignore-packages \"**/integration-tests/**\"",
-    "clean": "rimraf ./packages/*/lib",
+    "prebuild": "rimraf ./packages/*/lib",
     "lint": "eslint . --cache",
-    "postinstall": "yarn run clean --silent && yarn run build --silent",
+    "postinstall": "yarn build --silent",
     "precommit": "lint-staged",
-    "test": "jest",
+    "setup": "lerna clean --yes && lerna bootstrap",
```

Comment:
Hmm why are we running `lerna bootstrap` here? This should be using Yarn workspaces now to manage all node modules and package linking. Do you have Yarn v0.27.5 or newer locally?